### ISSUE TITLE
Dashboard on Library User Mode

### ISSUE THREAD

**Issue by alpacaShiro** [SE-Nex-Tech/Nex.Tech#92]
After logging out of the system, the user is redirected to the landing page in general user mode.
![image](https://github.com/SE-Nex-Tech/Nex.Tech/assets/100348938/49f8ae46-ff25-4c6b-b53e-85f0fc8108fa)

Upon clicking the "Back" button a few times, the user will be redirected to the database page, but the system still displays the navigation bar in general user mode.
![image](https://github.com/SE-Nex-Tech/Nex.Tech/assets/100348938/25f84036-3d74-4fed-a9e5-9cf4e5a58684)

Clicking the "Back" button again brought the user to the reports page.
![image](https://github.com/SE-Nex-Tech/Nex.Tech/assets/100348938/2a5e9fca-949a-4600-b71f-f12f435d8982)

Another "Back" button click brought the user to the dashboard page.
![image](https://github.com/SE-Nex-Tech/Nex.Tech/assets/100348938/48fafb75-7589-4517-8b62-ad791f968011)

Basically, the pages that the admin visits in the admin access mode can be accessed in general user mode after logging out of the admin access mode by clicking on the "Back" button.
